sdk/core/System.ClientModel/Directory.Build.targets

```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.targets))\Directory.Build.targets" />
+
+  <ItemGroup>
```

Comment:
Instead of removing this here I suggest setting these properties in your csproj https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.props#L154-L155

Reply:
I think we need both, no?  As this suggestion will set the properties for release, but the icon would still get pulled into the project by the common targets, no?